HealthcareSystem/PatientWebApp/wwwroot/js/add_feedback.js

```diff
@@ -0,0 +1,85 @@
+﻿$(document).ready(function () {
+
+	$('#add_feedback_form').submit(function (event) {
+
+		event.preventDefault();
+
+		var msg = $('#text_area_id').val();
+		var jmbg = "";
+		var name = "";
+		var surname = "";
+		var allowed = true;
+
+		if (!msg) {
+			$('#ap_error_msg').text('Enter a feedback message!');
+			$('#ap_error_msg').attr("hidden", false);
+
+			setTimeout(function () {
+				$('#ap_error_msg').attr("hidden", true);
+			}, 2000);
+
+			return;
+		}
+
+		//I take the first patient from the database, otherwise the currently logged in patient will be taken
```

Comment:
This makes sense since you can't log in a patient yet, but I'm not sure it's the most painless way to do it? Just taking the first patient on the backend instead might've been easier, that way you wouldn't have to expose an endpoint for getting a patient right now. Existing code will have to be changed later either way though, so whichever suits you more is good enough.